main.go

```diff
@@ -175,6 +175,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	if cn, err := getClusterName(); err == nil {
```

Comment:
Do we want to warn if we're unable to get cluster name or locality name? I think the previous revision was doing that.

Reply:
I think that's a nice thing to have. I've added it back